docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,151 @@
+
+# Widget Control
+
+The widget control provides information about installing and running widget applications.
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget applications
+
```

Comment:
Please change to:

- Sending update requests to the widget applications
- Retrieving details of running instance for the same package widget applications
Please avoid using "*" for bullet list, instead use "-"

Reply:
Done